app/src/pages/createLinkPage/createLinkPage.tsx

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+
+import { KibaException } from '@kibalabs/core';
+import { useNavigator } from '@kibalabs/core-react';
+import { Alignment, Box, Button, Direction, Form, InputType, PaddingSize, ResponsiveContainingView, SingleLineInput, Spacing, Stack, Text, TextAlignment } from '@kibalabs/ui-react';
+
+import { Domain, DOMAIN_ID_MAP } from '../../model';
+import { asyncSleep } from '../../util';
+
+export interface ICreateLinkPageProps {
+  domainId: string;
+}
+
+export const CreateLinkPage = (props: ICreateLinkPageProps): React.ReactElement => {
```

Comment:
do you wanna write the stubs for the e2e tests? one step towards better practices lol

Reply:
i'm planning on filling out the login ones soon

Reply:
added